src/resources/books.md

```diff
@@ -9,6 +9,15 @@ also cover Dart.
 If you find another Dart book that might be helpful, please
 [let us know.](https://github.com/dart-lang/site-www/issues)
 
+{{site.alert.warning}}
+  You may find other books published on the Dart programming 
+  language. We recommend that you check the date of publication, since 
+  Dart has evolved a great deal, and older books lack coverage of many 
+  important topics that are relevant to the modern Dart 2 language, 
+  including strong typing, null safety, FFI, the `dart` command line
+  utility, and our new developer tools. 
```

Comment:
Would it be more helpful to list the releases for important features, rather than publication dates? Maybe something like this:

```suggestion
  If you find a Dart book that isn't listed here,
  check which Dart SDK release it reflects.
  Older books lack coverage of Dart 2 topics such as
  strong typing (2.0), null safety (2.12),
  FFI, the `dart` command-line utility,
  and new developer tools. 
```

If not that, then maybe we should list the dates where each feature was introduced (e.g. 2.0 is Feb 2018, 2.12 is Mar 2021)... but books published after those dates might still have old information.

Reply:
For what it's worth, my feeling was that it essentially doesn't matter when those features were added. I picked them as examples of why any Dart 1 books are wildly out-of-date at this point. I like your other changes though!

Reply:
Maybe we should just say "make sure it was published recently"? Then keep my changes, but remove the release numbers.

Reply:
Oh, sure... It's your call, either way!